packages/components/CHANGELOG.md

```diff
@@ -38,6 +40,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0),
     ([#3542](https://github.com/porsche-design-system/porsche-design-system/pull/3542))
   - **Breaking Change**: `Select` and `Multi-Select` components no longer support native validation due to the removal
     of the underlying native `<select>` element.
+- `Pin-Code`: Remove native input and use ElementInternals API
```

Comment:
Does the Breaking Change from above also apply here? If yes also add disclaimer

Reply:
updated